impexp-client/src/main/java/org/citydb/gui/modules/exporter/preferences/CityGMLExportPreferences.java

```diff
@@ -37,13 +37,18 @@
 	public CityGMLExportPreferences(Config config) {
 		super(new CityGMLExportEntry());
```

Comment:
I thought about this as well. I kept "CityGML import" and "CityGML export" mainly because CityGML is a both a data model and an encoding. So, you have the "CityGML data model" and there are two encodings for that: 1) The "CityGML GML encoding" and 2) the "CityJSON encoding for CityGML". So, in a sense, "CityGML" remains a generic term for both encodings.

But then, on the other hand, this might be confusing for a user. So your comment makes a fair point and using "CityGML/CityJSON" might be more appropriate. Maybe we can have more input. @BWibo, @Son-HNguyen, what do you think?

Reply:
I get your point. So, now, I prefer the current name `CityGML Import` and `CityGML Export`. If we in the future add an additional preference node for the settings of delete operation, the name `CityGML Delete` looks much better than `CityGML/CityJSON Delete`, which looks confusing. 

If we decide to use `CityGML Import` and `CityGML Export`, then we may need to rename the `KML/COLLADA/glTF Export` to `VIS Export`, because the the name of preferences node now talks about _data model_, not _encoding or format_.

Reply:
Is the "VIS" accronym clear for all users? We could also use "Visualisation Export" as name for both the main tab and the preferences node. I'm not a native English speaker. In German, "Visualisierungsexport" sounds strange. A Google search gives me two hits when put in quotation marks... "Export für Visualisierung" only gets 6 hits. And, of course, you can also use CityGML exports for visualisation.

Reply:
Hm, I think the "VIS" should be OK. We have already used it for naming CLI operation. 
We could explain in the manual to say, "VIS" is the accronym of "Visualisation Models". Then the "VIS Export" will be interpreted as "Export of visualisation models" in English and "Export von Visualisierungsmodellen" in German.

Reply:
renamed with 4ff58625b36ba5b2a4b3bd56262c6c6528f54727. I think we could then also rename the `SPSHG` tab name of the spreadsheet plugin to `CSV Export`. And also the CLI command should be renamed to `export-csv`. This would be very consistent, and the current command name `spreadsheet-generate` is quite long anyway.

Reply:
The current spreadsheet generator also allows to export Excel format (.xlsx). So `CSV Export` seems not precise. But yes, the command name `spreadsheet-generate` somehow long. Maybe we could find a better name.

Reply:
Well, actually we only convert a CSV export to binary Excel format. So I would say CSV Export is quite appropriate. At least more precise than "spreadsheet", because we do not support/export any spreadsheet-specific additional functionality like functions, operations, filter, etc.

Anyway, I would like to merge the CityJSON export capabilities soon. Any more discussion related to this?

Reply:
I think this PR is ready for merging.

Reply:
Thanks, @yaozhihang. So, let's move the discussion about proper names for the Spreadsheet Plugin to this repository.